### DISCUSSION TITLE
How do I change the color of the focused file tabs bar?

### DISCUSSION THREAD

**Discussion by newk5** [apache/netbeans#5359] [Q&A] [answered]
![image](https://user-images.githubusercontent.com/32560510/214482872-0be11ec3-b5d4-456a-a6a2-2ea9590f04f4.png)

The right tab that contains the files opened is too dark in recent netbeans versions and I'd like to keep it at a ligher color or even the same color as when the code editor is not focused, as you can see on the screenshot I tried changing various properties but none of them worked. The unfocused tabs on the left side where you see "Projects", "Files" and "Services" is the type of color I'd like to have always even when focused.

**Comment by neilcsmith-net** [comment, marked as answer]
If I remember rightly, you want to look at `EditorTab.activeBackground`. Or one of the properties around it - https://github.com/apache/netbeans/blob/master/platform/o.n.swing.laf.flatlaf/src/org/netbeans/swing/laf/flatlaf/FlatDarkLaf.properties#L39

**Reply by newk5** [reply]
Thank you, that one worked.
